### DISCUSSION TITLE
Playbooks.ico issues

### DISCUSSION THREAD

**Discussion by Syke117** [Atlas-OS/Atlas#1215] [Help & Support]
everytime i remove this playbooks it still appearing when i restart my pc. how to remove this?

Please see attached image:
![image](https://github.com/user-attachments/assets/1862df1b-d999-4b5a-97fc-7ce1c6a1b25d)

**Comment by ekotin** [comment]
same

**Comment by cris2k47** [comment]
i made my own playbook and had the same problem, so its 100% AME fault

**Comment by kamild1996** [comment]
The workaround for now is to remove the Playbooks.ico file from C:\ProgramData\AME, then remove it from the desktop. It should no longer reappear on reboot.
